nevergrad/functions/functionlib.py

```diff
@@ -10,6 +10,8 @@
 from nevergrad.parametrization import parameter as p
 from nevergrad.common import tools
 from nevergrad.common.typetools import ArrayLike
+from sklearn.tree import DecisionTreeRegressor  # type: ignore
```

Comment:
I'd rather functionlib remains clear of external imports so that it can be imported without requiring more than the standard import. can this be moved to a separate module or package?